From 27827abc8e5c3c59741d7f6026e90ec705bbc574 Mon Sep 17 00:00:00 2001
From: Siddhant <siddhantsarkar2404@gmail.com>
Date: Tue, 27 Feb 2018 12:05:49 +0530
Subject: [PATCH] new function added user 2

---
 hello.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hello.c b/hello.c
index 2cd9c7f..ef239fe 100644
--- a/hello.c
+++ b/hello.c
@@ -1,5 +1,9 @@
 #include<stdio.h>
 void microkernel_sendmsg(char *);
+void newf()
+{
+	printf("This is a new function");
+}
 void main(){
 printf("Helloworld!\n");
 printf("This must be a monolithic design\n");
-- 
2.7.4

